roxie/udplib/udptrr.cpp

```diff
@@ -953,8 +953,6 @@ class CReceiveManager : implements IReceiveManager, public CInterface
         void noteDone(UdpSenderEntry *requester, const UdpRequestToSendMsg &msg)
         {
             const unsigned flowSeq = msg.flowSeq;
-            SendPermit * permit = requester->queryPermit(flowSeq);
```

Comment:
This was a change I stared at for a bit to try to convince myself that the warning was not hinting at buggy code. I _think_ it's just an unnecessary call.

Reply:
yes, I agree